resources/views/mgt/controller/index.blade.php

```diff
@@ -400,14 +400,26 @@ class="fa fa-envelope text-primary"></i></a></li>
                                             <li>Visiting:</li>
                                             @foreach ($user->visits()->get() as $visit)
                                                 <li>
-                                                    {{$visit->fac->id}} - {{$visit->fac->name}} - Mentor:
+                                                    {{$visit->fac->id}} - {{$visit->fac->name}} 
+                                                    <br> <span style="margin-left:2em;">MTR:
                                                     @if(\App\Classes\RoleHelper::isVATUSAStaff() || \App\Classes\RoleHelper::isFacilitySeniorStaff(\Auth::user()->cid, $visit->fac->id))
                                                         <a href="/mgt/controller/{{$user->cid}}/mentor/{{$visit->fac->id}}">
                                                             {{(\App\Classes\RoleHelper::isMentor($user->cid, $visit->fac->id))?"Yes":"No"}}
                                                         </a>
                                                     @else
                                                         {{(\App\Classes\RoleHelper::isMentor($user->cid, $visit->fac->id))?"Yes":"No"}}
                                                     @endif
+                                                    @if($user->rating>7)
```

Comment:
Should probably check RoleHelper::isInstructor here, for the same reasons as above.